### PR TITLE
add field descriptions for pg source and destination

### PR DESCRIPTION
**Pull request by cgardens**
## What
* Add descriptions to integration fields that did not yet have descriptions. These will be rendered in the UI to make the configuration process easier to understand.

### CODE REVIEW

File: dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/2168516a-5c9a-4582-90dc-5e3a01e3f607.json [hunk 1]
```diff
@@ -9,23 +9,29 @@
...
       "password": {
+        "description": "Password associated with the username.",
         "type": "string"
       },
       "dbname": {
+        "description": "Name of the database.",
         "type": "string"
       },
       "filter_dbs": {
+        "description": "This should be identical to dbname field.",
```
Comment (sherifnada) [inline dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/2168516a-5c9a-4582-90dc-5e3a01e3f607.json:34]: Is this going to be filled in from the UI? ideally not but fine if we need to do it for MVP

**Review by sherifnada — APPROVED** [review]

File: dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json [hunk 2]
```diff
@@ -10,7 +10,8 @@
     "properties": {
       "start_date": {
         "type": "string",
-        "description": "Start getting data from that date."
+        "description": "Start getting data from that date.",
+        "examples": ["YYYY-MM-DD"]
```
Comment (jrhizor) [inline dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json:14]: are examples also going to be shown on the UI?

Comment (jrhizor) [reply dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json:14]: also, should we use a regex to define the date format here?

**Review by jrhizor — APPROVED** [review]

File: dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json (hunk 2 above)
Comment (cgardens) [reply dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/37eb2ebf-0899-4b22-aba8-8537ec88b5a8.json:14]: > are examples also going to be shown on the UI?

yes. just added it to the update i posted in slack. luckily json schema supports this.

> also, should we use a regex to define the date format here?

yes. created an [issue](https://github.com/datalineio/dataline/issues/269). i think jsonschema supports this too actually. will handle separately from this PR.

File: dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/2168516a-5c9a-4582-90dc-5e3a01e3f607.json (hunk 1 above)
Comment (cgardens) [reply dataline-config/init/src/main/resources/config/SOURCE_CONNECTION_SPECIFICATION/2168516a-5c9a-4582-90dc-5e3a01e3f607.json:34]: yeah. i think we should do what you described but definitely out of scope of this PR.

**Pull request merged**